server/bleep/src/query/stopwords.rs

```diff
@@ -0,0 +1,79 @@
+// Portions of this code (the `phrases` function) are modifications of
+// https://github.com/yaa110/rake-rs/blob/master/src/rake.rs
+// licensed under the MIT License:
+/*
+Copyright (c) 2018 Navid Fathollahzade
+
+Permission is hereby granted, free of charge, to any
+person obtaining a copy of this software and associated
+documentation files (the "Software"), to deal in the
+Software without restriction, including without
+limitation the rights to use, copy, modify, merge,
+publish, distribute, sublicense, and/or sell copies of
+the Software, and to permit persons to whom the Software
+is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice
+shall be included in all copies or substantial portions
+of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF
+ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT
+SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR
+IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+*/
+
+use lazy_regex::regex;
+use once_cell::sync::OnceCell;
+use std::collections::HashSet;
+
+type StopWords = HashSet<String>;
```

Comment:
```suggestion
type StopWords = HashSet<&'static str>;
```
do we need these to be allocated strings?